### DISCUSSION TITLE
Refuse to load script because it violates the following Content Security Policy?

### DISCUSSION THREAD

**Discussion by 91d906h4** [community/community#23450] [New to GitHub]
<p>I didn’t use any external resources, and it’s still doesn’t work.<br>
Can someone tell me how to fix it?</p>
<p>Here is some screen shots of the error messages.<br>
<img src="https://user-images.githubusercontent.com/104009754/181077156-748c17e3-fc26-40aa-9a67-0930cdf03a72.png" alt="0" data-base62-sha1="kOeeuYAy083lVGiJFgsl3tmNJaW" width="491" height="102"></p>
<p><img src="https://user-images.githubusercontent.com/104009754/181077158-6182bc0c-8d3d-4a92-b7a9-ddc684c7f762.png" alt="1" data-base62-sha1="eVhm4Qa2E81WhZp25KH90vTrt3i" width="497" height="132"></p>
<p>And here is the code of my file.<br>
<img src="https://user-images.githubusercontent.com/104009754/181077159-7d9899d3-11e9-4e3d-83ba-efd73e316f82.png" alt="2" data-base62-sha1="j3DtPOuiBT0yKNWctmUoCMcBJZT" width="602" height="458"></p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>can you send the direct link of the repo ? that will be best so we can take a closer look too</p>

**Comment by 91d906h4** [comment]
<p>Thank you. I don’t know what was going on but this error disappeared the next day.</p>
